apps/docs/src/app/core/component-docs/button/examples/button-types-example.component.html

```diff
@@ -1,7 +1,7 @@
-<button fd-button label="Standard Button"></button>
-<button fd-button label="Emphasized Button" [fdType]="'emphasized'"></button>
-<button fd-button label="Ghost Button" [fdType]="'ghost'"></button>
-<button fd-button label="Positive Button" [fdType]="'positive'"></button>
-<button fd-button label="Negative Button" [fdType]="'negative'"></button>
-<button fd-button label="Attention Button" [fdType]="'attention'"></button>
-<button fd-button label="Transparent Button" [fdType]="'transparent'"></button>
+<button fd-button label="Standard Button" ariaLabel="Standard"></button>
+<button fd-button label="Emphasized Button" fdType="emphasized" ariaLabel="Emphasized"></button>
```

Comment:
those updates of the doc where `[fdType]` is replaced with `fdType` should be done in separate Pr

Reply:
@droshev Previously a comment was shared saying if we are touching any file and that file has unwanted braces can be removed and this change was introduced as part of the previously shared comment.

Reply:
I don't see why you would touch this file for the initial idea of your PR